test/api/Makefile

```diff
@@ -1,8 +1,8 @@
 all:
-	@$(CC) $(CFLAGS) -I../../src array.c ../../src/libsdb.a
-	@./a.out
-	@$(CC) $(CFLAGS) -I../../src refs.c ../../src/libsdb.a
-	@./a.out
+	@$(CC) -o array array.c $(CFLAGS) -I../../src array.c ../../src/libsdb.a
```

Comment:
what about using make targets for those instead of inlined cc calls?

Reply:
this will be fixed once I will rebase on top of PR #158